recipe/meta.yaml

```diff
@@ -23,47 +23,56 @@ requirements:
     - wheel
   run:
     - python
+    - arrow
     - binaryornot >=0.4.4
-    - click >=7.0
+    - click >=7.0,<9.0.0
     - jinja2 >=2.7,<4.0.0
-    - jinja2-time >=0.2.0
-    - poyo >=0.5.0
     - python-slugify >=4.0.0
+    - pyyaml >=5.3.1
     - requests >=2.23.0
-    - six >=1.10
+    - rich
 
 test:
+  imports:
+    - cookiecutter
+    - cookiecutter.main
+    - cookiecutter.environment
+    - cookiecutter.exceptions
   requires:
     - freezegun
     - git
-    # Omit to avoid falling back to Python 2.7
-    #- mercurial  # [not (win and py3k)]
+    - m2-grep  # [win]
+    # omit mercurial, because it's not available and optional
+    # - mercurial  # [not (win and py3k)]
     - pip
     - pytest
-    - pytest-cov
     - pytest-mock
-    - python 3.*
+    - python
   source_files:
     - tests
   commands:
-    - python -m pip check
+    - pip check
     - cookiecutter --version
```

Comment:
this line can be replaced with the next one.

Reply:
ok